docs/Containerfile.5.md

```diff
@@ -154,6 +154,49 @@ Current supported mount TYPES are bind, cache, secret and tmpfs.
 
               · rw, read-write: allows writes on the mount.
 
+**RUN --network**
+
+> Added in https://github.com/containers/buildah/pull/4566
```

Comment:
Not sure if that's needed. `git blame` would point to the commits.

Reply:
Removed it, my intention was for users to figure out if their release supports this or not, but its not important cause it will part of newer releases.